### PR TITLE
Cleaning and deleted unused file and commented code

### PR DESCRIPTION
**Pull request by SimonLab**
This PR is going to resolve the points in the [issue 61](https://github.com/FAC-GM/app/issues/61)

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab please assign the PR to someone when you're happy with it. :+1:

File: views/partials/pagination.html [hunk 1]
```diff
@@ -1,9 +1,5 @@
 <div class="pagination">
   <div class="pagination-center">
-  
-  {{{page_helper page page_url_prev page_url_next pages}}}      
-    <!-- <a class="newer-posts" id="prev" href="{{page_url_prev}}">&laquo;</a> -->
-     <!--  <span class="page-number">{{page}} &#47; {{pages}}</span>
-    <a class="older-posts" id="next" href="{{page_url_next}}">&raquo;</a> -->
+  {{{page_helper page page_url_prev page_url_next pages}}}
```
Comment (nelsonic) [inline views/partials/pagination.html]: the triple curly-brace *can* allow an XSS ... 
see: http://handlebarsjs.com/#html-escaping

**Comment by nelsonic** [comment]
Looks good. You're removing quite a *chunk* of old code. has it improved coverage? :grey_question:

**Comment by nelsonic** [comment]
@SimonLab please add a technical task to create a handlebar helper for that `{{{page_helper page page_url_prev page_url_next pages}}}` expression.
I'm happy to merge this now. :+1:

**Comment by SimonLab** [comment]
The coverage is at 100%, I'm not sure if codeclimate receive the coverage automatically

**Comment by SimonLab** [comment]
:+1: Thanks

**Pull request merged**